actions/cabal-ghc-compat/action.yml

```diff
@@ -0,0 +1,80 @@
+name: Check Cabal/GHC compatibility
+description: >
+  Check that the Cabal configuration file is compatible with the version of the
+  Cabal library bundled with GHC.
+inputs:
+  dirs:
+    description: Directories in which to run
+    required: false
+    default: "."
+  ghc:
+    description: GHC version, e.g., "9.10.1"
+    required: true
+  pkgs:
+    description: Extra packages to make available in the Nix shell
```

Comment:
This could use some explanatory examples so the user knows how to specify these.  "For example: `[ zlib, openssh ]`" or something like that.

Reply:
Noted that these are space-separated.